src/sentry/lang/native/applecrashreport.py

```diff
@@ -55,20 +57,22 @@ def _get_exception_info(self):
             if exc_name:
                 rv.append(
                     'Exception Codes: %s%s' %
-                    (exc_name, exc_addr is not None and (' at %s' % exc_addr) or '', )
+                    (exc_name, exc_addr is not None and (
+                        ' at %s' % exc_addr) or '', )
                 )
 
             if exception.get('thread_id') is not None:
                 rv.append('Crashed Thread: %s' % exception['thread_id'])
 
             if exception.get('value'):
-                rv.append('\nApplication Specific Information:\n%s' % exception['value'])
+                rv.append('\nApplication Specific Information:\n%s' %
+                          exception['value'])
 
         return '\n'.join(rv)
 
     def get_threads_apple_string(self):
         rv = []
-        for thread in self.threads or []:
+        for thread in (self.exception + self.threads) or []:
```

Comment:
This can be `None` + `[]` which will fail (or the other way round).

Reply:
Also since this is now different objects i would at least not call it thread (code now assumes either thread or exception).